src/Redux/SuperAdmins/thunks.js

```diff
@@ -0,0 +1,122 @@
+import {
+  getSuperadminsPending,
+  getSuperadminsError,
+  getSuperadminsSuccess,
+  addSuperadminPending,
+  addSuperadminError,
+  addSuperadminSuccess,
+  editSuperadminPending,
+  editSuperadminError,
+  editSuperadminSuccess,
+  deleteSuperadminPending,
+  deleteSuperadminError,
+  deleteSuperadminSuccess,
+  resetPrimaryStates
+} from './actions';
+
+import { handleDisplayToast, setContentToast } from '../Shared/ResponseToast/actions';
+const responseInfo = { resState: '', resMessage: '', superadmins: [] };
```

Comment:
I saw this and I searched if you are using it but no, only in the error actions, but after that payload is unused. I think you can remove this.